mozjs/js/rust/src/jsslot.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function String_numslot() {
+    return 1;
+}
+
+function Vec_numslot() {
+    return 3;
+}
```

Comment:
I don't know what till has in mind though. It's pretty crowded in the all hands. I might miss something.